returnn/hdf.py

```diff
@@ -309,12 +310,25 @@ def __init__(
         self.allophone_file = allophone_file
         self.state_tying_file = state_tying_file
         self.out_hdf_files = [self.output_path(f"data.hdf.{d}", cached=False) for d in range(len(alignment_caches))]
```

Comment:
```suggestion
        self.out_hdf_files = [self.output_path(f"data.hdf.{d}") for d in range(len(alignment_caches))]
```

Reply:
I think you missed the line below I was referring to :D